src/main/kotlin/lotto/controller/LottoProgramStart.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.controller
+
+import lotto.model.LottoNumberMaker
+import lotto.model.MatchCalculator
+import lotto.model.PurchaseAmountParser
+import lotto.model.ReturnRateCalculator
+import lotto.view.OutputView
+
+object LottoProgramStart {
+    fun run() {
+        val purchaseAmount = PurchaseAmountHandler.readValidated()
+        val ticketCount = PurchaseAmountParser.parse(purchaseAmount)
+        OutputView.printTicketCount(ticketCount)
+
+        val lottoNumber = LottoNumberMaker.makeTotalTicket(ticketCount)
+        LottoNumberMaker.printTotalTicket(lottoNumber)
+
+        val winningNumbers = WinningNumbersHandler.readValidated()
+        val bonusNumber = BonusNumberHandler.readValidated(winningNumbers)
+
+        val winningAmount = MatchCalculator.calculate(lottoNumber, winningNumbers, bonusNumber)
+        val returnRate = ReturnRateCalculator.run(purchaseAmount, winningAmount)
+
+        OutputView.printWinningStatisticsStartLine()
+        OutputView.printWinningStatistics()
+        OutputView.printReturnRate(returnRate)
```

Comment:
It might be better to keep just one method here for clarity.
Using a single, unified method could make the code easier to maintain!